### DISCUSSION TITLE
manually clear the frontend cache

### DISCUSSION THREAD

**Discussion by giuseppemari75** [kalkih/mini-graph-card#1088] [Q&A] [answered]
![image](https://github.com/kalkih/mini-graph-card/assets/149202186/98105ad7-2089-48c7-b864-11ffd11e7a02)
HI, please someone knows how-to do this ?
"You need to manually clear the frontend cache after updating."
I've updated but I don't know how to manually clear the frontend cache
thanks a lot

**Comment by ildar170975** [comment, marked as answer]
1. If you use a HA Companion App: go to Settings for the App -> Find a "reset frontend cache" button.

2. If you use a browser - it depends on a particular browser. Google "Chrome clear cache", put your browser instead of "Chrome".
